### PR TITLE
MM-18028 - Document how to set the max file size in the clamav docker image.

### LINKED ISSUE #11999: Document how to set the max file size in the clamav docker image.
The default max file size for clamav is lower then the default max file size for Mattermost. This leads to a lot of reported issues where the antivirus plugin will reject files that are larger then 25MB, the default max size for clamav.  

Documentation should be created and added to the installation process to increase the max file size in the clamav docker container to match the max file size in Mattermost. Ideally this would be as simple as possible. 

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-18028

### PR DESCRIPTION
**Pull request by Francois-D**
#### Summary
Document how to set the max file size parameter in the clamav docker image.
Mattermost default size is 50 MB the default for ClamAV is 25MB.  Modified documentation on how to set the default ClamAV size to match Mattermost.

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/11999

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ Currently the plugin supports [ClamAV anti-virus software](https://www.clamav.ne
 
 1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-antivirus/releases) and download the latest release for your Mattermost server.
 2. In the Mattermost System Console under **System Console > Plugins > Plugin Management** upload the file to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with Docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and set up a server with an open port at 3310:
+3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with Docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and set up a server with an open port at 3310: and MaxFileSize of 50 MB (Mattermost Default).
```
Comment (hanzei) [inline README.md]: Could you please note that the value for `MaxFileSize` should be updated when the Mattermost Setting `MaxFileSize` gets updated and link to https://docs.mattermost.com/administration/config-settings.html#maximum-file-size?

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for helping with the documentation :+1:

File: README.md (hunk 1 above)
Comment (Francois-D) [reply README.md]: Great suggestion, I've updated  the readme.

**Review by hanzei — APPROVED** [review]
:tada:

**Review by iomodo — APPROVED** [review]
LGTM! @Francois-D, thanks for your contribution!

**Comment by Francois-D** [comment]
Hello @aaronrothschild 
Just checking on the status of this issue.  Please let me know if my changes are acceptable.

Thanks in advance

**Comment by Francois-D** [comment]
Hello @hanzei 
The PR for the CODEOWNERS removes the 2nd maintainer from the list.  
https://github.com/mattermost/mattermost-plugin-antivirus/pull/17
Should this also be reflected in this PR as well?
Or should I create a new issue.
Thanks.

File: README.md [hunk 2]
```diff
@@ -18,10 +18,9 @@ Currently the plugin supports [ClamAV anti-virus software](https://www.clamav.ne
...
-
+3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with Docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and set up a server with an open port at 3310: and MaxFileSize of 50 MB. Mattermost's MaxFileSize default value is subject to change. To ensure that the correct value is set, verify your value at the following link: [Maximum File Size](https://docs.mattermost.com/administration/config-settings.html#maximum-file-size).  
    ```
-   docker run -d -p 3310:3310 mkodockx/docker-clamav
+   docker run -e CLAMD_CONF_MaxFileSize=50M -d -p 3310:3310 mkodockx/docker-clamav
    ```
 
 4. Once clamd server is running, configure the plugin to make requests to your clamd instance. Go to **System Console > Plugins > Antivirus** and configure **Clamav Host and Port** to point at your clamd instance.
```
Comment (aaronrothschild) [inline README.md:26]: @Francois-D I JUST ran intot his with someone else...it's not obvious what the clamAV hostname is that should be put into the "ClamAC Host and Port".  For example, someone put in "localhost:3310" to point to clamAV, but that won't work.  Would the default docker hostname for the ClamAV docker image be "docker-clamav:3310"?  If so, can we insert that as an example so it's clearer to admins on setup?

**Review by aaronrothschild — APPROVED** [review]
Thanks for helping out @Francois-D

File: README.md (hunk 2 above)
Comment (Francois-D) [reply README.md:26]: @aaronrothschild 
The documentation should have a step 4 which checks if the docker container was launched correctly.  The command is _docker ps -a_

Screen Shot 2020-04-29 at 1 19 53 AM

The new step 5 would be to configure the **Clamav Host and Port**  .  

Mattermost does not function when using 'localhost' in the clamav configuration.  In order to function the _hostname_ of the Docker host should be set. In my case the docker host was the same as the Mattermost Server.  To retrieve the hostname of the docker host run the command in a terminal window: _hostname -s_ on the docker host.

As the configuration is pre populated with the value _localhost_:_port_.The port number that was set during install was set correctly   I recommend  that the localhost value be set to the hostname of the server.  Let me know your thoughts.  Thanks in advance.
Screen Shot 2020-04-28 at 7 23 15 PM

**Pull request merged**